playwright/e2e/crypto/utils.ts

```diff
@@ -292,17 +292,31 @@ export async function doTwoWaySasVerification(page: Page, verifier: JSHandle<Ver
 }
 
 /**
- * Open the security settings and enable secure key backup.
+ * Open the encryption settings and enable the key backup by
+ * - enabling key storage
+ * - set up a recovery key
  *
- * Assumes that the current device has been cross-signed (which means that we skip a step where we set it up).
+ * Assumes that the current device has been verified
  *
  * Returns the recovery key
  */
 export async function enableKeyBackup(app: ElementAppPage): Promise<string> {
```

Comment:
I think this should probably be renamed `enableRecovery`, but that doesn't have to happen right now